go/apps/api/openapi/spec/paths/v2/permissions/listRoles/index.yaml

```
@@ -3,7 +3,8 @@ post:
     - permissions
   summary: List roles
   description: |
-    Retrieve all roles in your workspace including their assigned permissions. Results are paginated and sorted alphabetically by role name.
+    Retrieve all roles in your workspace including their assigned permissions. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)